code/CMSMenu.php

```diff
@@ -379,7 +379,7 @@ public static function get_cms_classes($root = null, $recursive = true, $sort =
         if (!$root) {
             $root = LeftAndMain::class;
         }
-        $abstractClasses = [LeftAndMain::class, CMSMain::class];
+        $abstractClasses = [LeftAndMain::class];
```

Comment:
This change is needed so that `CMSMain` can be included in the left menu and have its route built.
Without this, it was the `CMSPagesController` which was a very-nearly-useless subclass of `CMSMain` that was used.
That class has been merged into `CMSMain` so we need its menu item back